charts/kafka-operator/values.yaml

```diff
@@ -6,7 +6,7 @@ replicaCount: 1
 
 # Lists the secrets you need to use to pull kafka-operator image from a private registry.
 imagePullSecrets: []
```

Comment:
I think we also can use this here:
imagePullSecrets:
or 
# imagePullSecrets:

This way we dont need to use [] which can be mislead

Reply:
Good idea, thanks!